components/src/core/drawer/drawer-nav-item.tsx

```diff
@@ -182,7 +186,9 @@ export const DrawerNavItem: React.FC<DrawerNavItemProps> = (props) => {
     const previousActive = usePrevious(activeItem || '');
 
     // approximating primary[200] but we don't have access to it directly from the theme
-    const lightenedPrimary = color(theme.colors.primaryBase || theme.colors.primary)
+    const lightenedPrimary = color(
```

Comment:
blue[200] is available in the theme properties now isn't it? Or was that a one-off that we hardcoded? Looks like maybe there is a use case for defining it after all...

Reply:
This is a one-off override in the light theme. We can get blue[200] from the dark theme though. I can add a one-off to the light theme for this or we can leave it as is...

Reply:
we can leave it for now